kratos/utilities/mortar_utilities.h

```diff
@@ -394,36 +392,36 @@ class MortarUtilities
     
     /**
      * This function gives you the indexes needed to order a vector 
-     * @param vect: The vector to order
-     * @return idx: The vector of indexes
+     * @param ThisVector The vector to order
+     * @return idx The vector of indexes
      */
     
     template <typename TType>
-    static inline std::vector<std::size_t> SortIndexes(const std::vector<TType> &vect) 
+    static inline std::vector<std::size_t> SortIndexes(const std::vector<TType> &ThisVector) 
```

Comment:
i confirm my comment. this should not be "inline"

Reply:
Why not? , it is a small function

Reply:
your code is small, but you are calling sort, which is not a small one

Reply:
Ok

Reply:
jeje you still have the inline